src/main/resources/db/changelog/data/financialTransaction.csv

```diff
@@ -0,0 +1,9 @@
+id,wallet_id,transaction_type,amount,transaction_date,description
+1,1,INCOME,15000,2022-02-19T09:47:52.595721658,""
+2,2,EXPENSES,5000,2022-02-19T09:47:52.595721658,null
```

Comment:
How to paste in proper way `null` value in CSV file? How CSV or Liqubase config (?) know it's not just a String value "null" ?